### PR TITLE
Some updates in utils package.

### PR DESCRIPTION
**Pull request by emiryazicii**
Team members , please review my changes and leave some comments. Thanks.

### CODE REVIEW

File: src/utils/StringUtils.java [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
 public class StringUtils {
+
+    public static String concatenateStrings(String str1, String str2) {
+        return str1 + str2;
+    }
+
+    public static boolean isNullOrEmpty(String str) {
+        return str == null || str.isEmpty();
+    }
+
```
Comment (jika17) [inline src/utils/StringUtils.java:12]: public static int frequency(String sentence, String word) {
        int count = 0;
        while (sentence.contains(word)){
            count ++;                     
            sentence = sentence.replaceFirst(word,""); 
        }
        return count;    
    }

**Comment by bataborisa** [comment]
I think those methods a Great! We can add some more, so everyone can have benefits!

**Comment by emiryazicii** [comment]
Thanks for your feedback , Boris. We set a meeting to discuss about it . It will be on wednesday.

**Review by bataborisa — COMMENTED** [review]
Now we are talking! :) Great Job Emir!

**Review by drnajimov — APPROVED** [review]
Looks good

**Pull request merged**